docs/pages/careers/developer-advocate.md

```diff
@@ -1,10 +1,10 @@
-# Developer Advocate
+# Developer Advocate / DevEx Engineer
 
-<p class="description">You will build a thriving and connected developer community around our suite of products.</p>
+<p class="description">You will delight our users and make them excited about the product roadmap.</p>
 
 ## Details of the Role
 
-- **Location**: Remote (strong requirement for UTC-8 to UTC+2).
+- **Location**: Remote (strong requirement for UTC-6 to UTC+2).
```

Comment:
```suggestion
- **Location**: Remote (strong preference for UTC-6 to UTC+2).
```

If it's a "strong requirement", I reckon we could just say something like "_only for..._". Otherwise, I think "_preference_" makes more sense! 😬